tests/components/brottsplatskartan/test_config_flow.py

```diff
@@ -0,0 +1,209 @@
+"""Test the Brottsplatskartan config flow."""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.brottsplatskartan.const import CONF_AREA, DOMAIN
+from homeassistant.const import CONF_LATITUDE, CONF_LOCATION, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] == {}
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_AREA: "none",
+        },
+    )
+    await hass.async_block_till_done()
+
+    assert result2["type"] == "create_entry"
+    assert result2["title"] == "Brottsplatskartan HOME"
+    assert result2["data"] == {
+        "area": None,
+        "latitude": None,
+        "longitude": None,
+        "app_id": "ha-1234567890",
+    }
```

Comment:
I think HOME shouldn't be different from specific location, and it should store hass.config.longitude/hass.config.latitude